homeassistant/components/zwave_js/sensor.py

```diff
@@ -164,7 +179,7 @@ def state(self) -> Optional[str]:
             not str(self.info.primary_value.value)
             in self.info.primary_value.metadata.states
         ):
-            return None
+            return str(self.info.primary_value.value)
```

Comment:
What would a value be eg in this case?

Reply:
there are notification CC values that don't have states, just numeric values. The state in this case would be that numeric value

Reply:
Example:
```json
				}, {
					"endpoint": 0,
					"commandClass": 113,
					"commandClassName": "Notification",
					"property": "alarmType",
					"propertyName": "alarmType",
					"ccVersion": 1,
					"metadata": {
						"type": "number",
						"readable": true,
						"writeable": false,
						"min": 0,
						"max": 255,
						"label": "Alarm Type"
					},
					"value": 12
				}, {
```

Reply:
Ok, thanks.